rerun_py/rerun_sdk/rerun/archetypes/asset3d_ext.py

```diff
@@ -50,4 +54,6 @@ def from_bytes(blob: bytes, media_type: MediaType | None) -> Asset3D:
         from . import Asset3D
 
         # TODO(cmc): we could try and guess using magic bytes here, like rust does.
-        return Asset3D(blob=blob, media_type=media_type)
+        with catch_and_log_exceptions(context="Asset3D.from_file"):
+            return Asset3D(blob=blob, media_type=media_type)
+        return Asset3D._clear()
```

Comment:
Here you use `_clear()`, but not everywhere. I'm confused

Reply:
`_clear()` currently *returns* a new `Asset3D()`, whereas the other functions (inside of an existing `__init__`) need to *modify* the current object to put it into a cleared state.